src/frontend.rs

```diff
@@ -244,7 +271,7 @@ pub struct NodeSpec {
 #[derive(Debug, PartialEq)]
 pub struct RelSpec {
     slot: usize,
-    rel_type: Token,
+    rel_type: RelType,
```

Comment:
Do you think this could be an `Option`? Eg. when you `CREATE` stuff you need to specify the reltype (at least in regular cypher, maybe not in GQL if they are talking about multi-reltype support there..). So `(a)-->(b)` is in `MATCH`, and in that case I think `Option` captures the situation better; highlighting that there was no rel predicate specified at all?